packages/orchestration/src/orchestration-api.ts

```diff
@@ -193,8 +193,9 @@ export interface OrchestrationAccountCommon {
    * Transfer an amount to another account, typically on another chain.
    * The promise settles when the transfer is complete.
    * @param amount - the amount to transfer. Can be provided as pure data using denoms or as ERTP Amounts.
-   * @param destination - the account to transfer the amount to.
-   * @param [opts] - an optional memo to include with the transfer, which could drive custom PFM behavior, and timeout parameters
+   * @param destination - the account to transfer the amount to. The denom in the destination arg has to be 
+   * registered with `chainHub` via {@link `registerAsset`} API.
```

Comment:
This link doesn't seem to work.

you might try `{@link ChainHub}`, after checking that ChainHub is imported.
Hm. It might not be imported. It's sort of a lower level concept than OrchestrationAccount.
I should think this over a bit.